bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/seq/EdIteratorFolder.java

```diff
@@ -80,12 +89,22 @@ public JPanel getWindow() {
     /** Set the data object corresponding to this editor. */
     @Override public void init() {
         final ISPObservation obs = getContextObservation();
-        // No context observation if this sequence component is in a conflict
-        // folder!
+        // No context observation if this sequence component is in a conflict folder!
         if (obs != null) {
             obs.addCompositeChangeListener(obslogUpdater);
         }
 
+        // show/hide ITC result panels depending on instrument
+        final ISPObsComponent instrument = getContextInstrument();
+        if (instrument != null) {
+            final SPComponentType type = instrument.getType();
+            _w.tabbedPane.remove(itcImagingPanel.peer());
+            _w.tabbedPane.remove(itcSpectroscopyPanel.peer());
+            if (itcImagingPanel.visibleFor(type))      _w.tabbedPane.add("ITC Imaging",      itcImagingPanel.peer());
+            if (itcSpectroscopyPanel.visibleFor(type)) _w.tabbedPane.add("ITC Spectroscopy", itcSpectroscopyPanel.peer());
+        }
```

Comment:
True, with ```null``` it should remove them, however I think that "should" never happen, but better safe than sorry.

In almost all cases it is either imaging or spectroscopy. In very few cases it can be mixed. It would be great to only show the panel that's actually needed (non-emtpy sequence), I'll check if there is a simple way to do that.